.travis.yml

```diff
@@ -3,12 +3,12 @@ cache: yarn
 script:
   - yarn --version
   - yarn check --integrity
-  - yarn setup
+  - yarn copy-assets
```

Comment:
Presence of compiled JS files in the same location as TS files broke coverage report. We do not need to compile TS into JS on travis so this is quick fix. We might need to look into that issue carefully very soon as this is another issue caused by having TS and JS in the same place.

Reply:
doesn't the same problem happen on jenkins with sonar? and it will be happening on CNP as well I assume @rlewan ?

Reply:
I removed TS compilation from Jenkinsfile as well. You were right @timja-kainos , the same issue was with sonar reporting lower coverage.

Reply:
Yes, sonar was scanning `*.js` files.

Reply:
@rlewan Does this need to be changed in CNP branch as well so that it is 
```
afterCheckout
yarn copy-assets

afterSonar (something like that)
yarn compile
```

Reply:
I would postpone changes to CNP till we decide whether we need to compile into JS. When https://tools.hmcts.net/jira/browse/ROC-2908 is played I plan to either remove TS>JS compilation and stick back to ts-node (if it is fast enough) or use separate TS>JS compilation target directory.

Reply:
👍